taprpc/mintrpc/mint.proto

```diff
@@ -187,6 +187,20 @@ message MintAsset {
     a BIP-86 key will be derived from the underlying wallet.
     */
     taprpc.ScriptKey script_key = 12;
+
+    /*
+    DecimalDisplay dictates the number of decimal places to shift the amount to
```

Comment:
nit: either use just "Decimal display" or the actual field name "decimal_display" at beginning of sentence.